src/components/Delegates/DelegatesTabs/DelegatesTabs.tsx

```diff
@@ -73,6 +81,20 @@ export default function DelegateTabs({ children }: { children: ReactNode }) {
           {hasIssuesFilter && tabParam !== "citizens" && <IssuesFilter />}
           {hasEndorsedFilter && tabParam !== "citizens" && <EndorsedFilter />}
           {tabParam === "citizens" ? <CitizensFilter /> : <DelegatesFilter />}
+          {value === "delegates" && (
+            <div className="flex items-center gap-2 bg-wash rounded-full px-4 py-2">
+              <button onClick={() => setLayout("grid")}>
+                <LayoutGrid
+                  className={`h-6 w-6 ${layout === "grid" ? "text-secondary fill-secondary" : "text-secondary/30 fill-secondary/30"}`}
+                />
+              </button>
+              <button onClick={() => setLayout("list")}>
+                <AlignJustify
+                  className={`h-6 w-6 ${layout === "list" ? "text-primary" : "text-secondary/30"}`}
+                />
+              </button>
```

Comment:
I'm not familiar with these components, I'd prefer if we could write regular html elements and style them with tailwind. I don't really like utility components like <AlignJustify /> which I'm guessing just positions content a certain way. Something we could do with tailwind.

We had something similar `<HStack />` and `<VStack/>` which only rendered a div with flex and flex-col or flex-row and I think it ends up being more challenging when you have another layer of abstraction via a component that is only adding a few classes.

Sorry to be a pest on this!